markdown/events/2021/bytesize-4-github-contribution-basics.md

```diff
@@ -34,3 +34,97 @@ This will cover:
+
+I selected myself, then things are running for a couple of seconds and then you will see that there is a link to copy made, which is also listed in the main repository here as a fork of the original repository in nf-core you can actually start working on. [11:08](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=668)
+
+So after we made that fork, we can make a local clone of that to work on this pipeline so we have two versions of the repository, the nf-core one and our own and now we can work on adding bits here and there. So the first step we will have to do then is remember the basic Git introduction. We have to clone it, download it to our local machine. So what I do here is hit clone and I type the url; it takes a couple of seconds depending on the pipeline size and then you hopefully have a local copy of that pipeline available. That of course also requires an installed Git version after that I can just use an editor or whatever other tool I have at hand, and change some stuff in the code and simply follow this approach after adding these local changes. So I'm making local changes, then we add the changed files for example if I added a new let's say a document or a text file or an image or something like that then I can simply add that to the repository by using Git art.
+I commit these changes with a nice little comment message. So for example if I add some images I should probably have some nice comment message talking about like I've added images that are used for documentation;  that would be a message where people later also would be able to understand okay what has been done in this specific comment in the past. And then I push these changes to my repository so that I just do a git push and this will actually copy the changes back to my own repository which is on Github. [12:22](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=742)
```

Comment:
```suggestion
I commit these changes with a nice little comment message. So for example if I add some images I should probably have some nice commit message talking about like I've added images that are used for documentation;  that would be a message where people later also would be able to understand okay what has been done in this specific commit in the past. And then I push these changes to my repository so that I just do a git push and this will actually copy the changes back to my own repository which is on Github. [12:22](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=742)
```